scripts/esbuild.js

```diff
@@ -1,8 +1,40 @@
 /* eslint-disable no-console */
 const esbuild = require('esbuild-wasm');
+const fs = require('node:fs');
+const path = require('node:path');
+
+require('dotenv').config({ path: path.resolve(__dirname, '../.env.local') });
 
 const production = process.argv.includes('--production');
 const watch = process.argv.includes('--watch');
+const { DHC_PACKAGES_PATH } = process.env;
+
+const optionalPlugins = [];
+
+// Alias @deephaven/* packages to custom path
+if (DHC_PACKAGES_PATH != null) {
+  if (!fs.existsSync(DHC_PACKAGES_PATH)) {
+    throw new Error(`DHC packages path ${DHC_PACKAGES_PATH} does not exist.`);
+  }
+
+  console.log('Aliasing @deephaven/* packages to:', DHC_PACKAGES_PATH);
+
+  optionalPlugins.push({
+    name: 'dh-alias-plugin',
+    setup(build) {
+      build.onResolve({ filter: /^@deephaven\/.*/ }, args => {
```

Comment:
Curious why it's being done this way instead of the `resolve.alias` method we use in `iris` repo?

Reply:
Ah my bad. Silly me.